nth-prime/nth_prime.rb

```diff
@@ -0,0 +1,20 @@
+class Prime
+  def self.primes?(integer)
+    (2..integer/2).each {|x| return false if (integer % x) == 0 }
```

Comment:
The code is hard to read. Have you considered to make it readable by rename variables?